gulpfile.js

```diff
@@ -1476,7 +1477,8 @@ ${source}
     const assignmentName = basename(file, extname(file));
     if (publicModules.has(assignmentName)) {
       publicModules.delete(assignmentName);
-      source += `declare module "cesium/Source/${moduleId}" { import { ${assignmentName} } from 'cesium'; export default ${assignmentName}; }\n`;
+      //PROPELLER HACK
+      source += `declare module "cesium/Source/${moduleId}" { import { ${assignmentName} } from '@propelleraero/cesium'; export default ${assignmentName}; }\n`;
```

Comment:
It looks like the first `@propelleraero/` is missing here.

https://github.com/PropellerAero/cesium/blob/main/gulpfile.cjs#L1283

Reply:
Not quite sure i understand correctly where we want to change

Reply:
```
source += `declare module "cesium/Source/${moduleId}" ...
vs
source += `declare module "@propelleraero/cesium/Source/${moduleId}" ...
```